portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+  
```

Comment:
remove this space, have you run the clang-format ?

Reply:
I don't think this space needs to be removed, according to clang-format result. But thank you for viewing that! I have just adjusted all files in step/portfolio again. :) And make changes for the space you pointed below.